libs/anglify/src/modules/snackbar/snackbar.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectionStrategy, Component, HostBinding, Inject } from '@angular/core';
+import { SnackbarContext } from './snackbar-context.interface';
+import { SNACKBAR_CONTEXT } from './snackbar.service';
+
+@Component({
+  selector: 'anglify-snackbar',
+  templateUrl: './snackbar.component.html',
+  styleUrls: ['./snackbar.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SnackbarComponent {
+  public constructor(@Inject(SNACKBAR_CONTEXT) public readonly context: SnackbarContext) {}
+
+  @HostBinding('class')
+  protected get classList() {
+    const classNames: string[] = [this.context.position];
+
+    if (this.context.stacked) {
+      classNames.push('stacked');
```

Comment:
The stacked class is added to the classList here, but there is no stacked style in the stylesheet. Is this intended?

Reply:
Nope, that is actually missing now that you mention it, oops, will fix.